### PR TITLE
Control Left-Right Arrow Shortcuts [API-1208]

### PR DESCRIPTION
**Pull request by utku-caglayan**
Implements shortcuts to jump words on prompt. Documents already existing shortcuts on Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,33 @@ hzc --help
 # Non-interactive mode
 hzc map --name myMap put --key myKey --value myValue
 ```
+### Keyboard Shortcuts
+Hazelcast CLC inherits following shortcuts from [c-bata/go-prompt](https://github.com/c-bata/go-prompt)
```
Comment (yuce) [inline README.md:66]: I think there's no need to add a reference to that link for the shortcuts. Let's add all of the supported shortcuts in a separate PR.

File: README.md [hunk 2]
```diff
@@ -62,6 +62,33 @@ hzc --help
...
+<kbd>Ctrl + N</kbd>  | Next command (Down arrow)
+<kbd>Ctrl + F</kbd>  | Forward one character
+<kbd>Ctrl + B</kbd>  | Backward one character
+<kbd>Ctrl + D</kbd>  | Delete character under the cursor
+<kbd>Ctrl + H</kbd>  | Delete character before the cursor (Backspace)
+<kbd>Ctrl + W</kbd>  | Cut the word before the cursor to the clipboard
+<kbd>Ctrl + K</kbd>  | Cut the line after the cursor to the clipboard
+<kbd>Ctrl + U</kbd>  | Cut the line before the cursor to the clipboard
+<kbd>Ctrl + L</kbd>  | Clear the screen
+With few additions:
```
Comment (yuce) [inline README.md:84]: How about adding a blank line before this one?

File: README.md [hunk 3]
```diff
@@ -62,6 +62,33 @@ hzc --help
...
+
+Key Binding          | Description
+---------------------|---------------------------------------------------------
+<kbd>Ctrl + C</kbd>  | Cancel running command or close the app
```
Comment (yuce) [inline README.md:88]: It closes the app when Ctrl+C is pressed twice, doesn't it? If so, the description may be updated to reflect that.

File: README.md [hunk 4]
```diff
@@ -62,6 +62,33 @@ hzc --help
...
+<kbd>Ctrl + →</kbd>  | Go to the end of to next word
```
Comment (yuce) [inline README.md:89]: The arrow character is displayed really small in my browser. How about `->` instead? Or `-> (right arrow)`?

File: README.md [hunk 5]
```diff
@@ -62,6 +62,33 @@ hzc --help
...
+<kbd>Ctrl + ←</kbd>  | Go to the start of the previous word
```
Comment (yuce) [inline README.md:90]: The same comment for right arrow.

**Review by yuce — CHANGES_REQUESTED** [review]
A few minor comments.

File: README.md (hunk 3 above)
Comment (utku-caglayan) [reply README.md:88]: Pressing once is sufficient for prompt mode. If you have a long running command in progress, you need to press twice to first cancel the command, second to exit the prompt. I think it is understood.

File: README.md (hunk 2 above)
Comment (utku-caglayan) [reply README.md:84]: Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/44/commits/9c8793bb9e42a52b4ba2005d574530a43b8d9a69

File: README.md (hunk 4 above)
Comment (utku-caglayan) [reply README.md]: Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/44/commits/9c8793bb9e42a52b4ba2005d574530a43b8d9a69

File: README.md (hunk 5 above)
Comment (utku-caglayan) [reply README.md]: Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/44/commits/9c8793bb9e42a52b4ba2005d574530a43b8d9a69

**Review by yuce — APPROVED** [review]
Looks good.

**Comment by emreyigit** [comment]
LGTM

**Pull request merged**